components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -190,16 +190,18 @@ const HostDashboardExpenses = ({ accountSlug: hostSlug, isDashboard }) => {
         <h1 className="text-2xl font-bold leading-10 tracking-tight">
           <FormattedMessage id="Expenses" defaultMessage="Expenses" />
         </h1>
-        <SearchBar
-          height="40px"
-          defaultValue={query.searchTerm}
-          onSubmit={searchTerm =>
-            router.push({
-              pathname: pageRoute,
-              query: getQueryParams({ searchTerm, offset: null }),
-            })
-          }
-        />
+        <div className="w-[276px]">
```

Comment:
The icon doesn't seem to appear on Dashboard > Expenses, nor for most dashboard pages. It's likely because they're not using `value` but `defaultValue`. Could you make sure to test these? It's ok to not support the icon in a few places, but then we shouldn't touch the files

Reply:
In this case `defaultValue` is correct. This component is related to admin > expenses.

I found out that in Dashboard > expenses, we are using a `Filterbar.tsx` component. This component utilizes a `filter.StandaloneComponent`. Until then, I hadn't noticed that this search component existed. This component employs a slightly different approach. I believe this component never had a clear input button.

If you'd like, we can try to enhance/improve this component. What do you think?

Reply:
My bad I got confused between the new dashboard and the old admin, we really need to get rid of those components.

It's all good then, let's ship this!